chore: [IA-32] Open ToS links in external browser

Undermaken opened the pull request:
## Short description
This PR allows to open links, contained in the Terms of Service web page, in the device browser.
It injects a JS into the webview to intercept the clicked links and it command to open them in the system default browser.

This PR also fixes a navigation bug for which if the user opens an internal link he/she can't go back in the previous web page.

### changes
- add the `link clicked` handling
- remove the `end of scroll` handling since it is never sent by the webview (no script added)

### before
https://user-images.githubusercontent.com/822471/122024995-3d918680-cdc9-11eb-98d4-e32f2fa414e3.mov

### now

https://user-images.githubusercontent.com/822471/122025040-4aae7580-cdc9-11eb-90a5-28934d52afd8.mov

CrisTofani reviewed (changes requested):
The PR lgtm, since we removed the scrollend handler, what about to clean up the state definition removing the information?

Undermaken commented:
> The PR lgtm, since we removed the scrollend handler, what about to clean up the state definition removing the information?
removed with commit https://github.com/pagopa/io-app/pull/3142/commits/cc880ff452b25e6c8badc5d48dad3ddd613427a1
